packages/design-system-tokens/lib/utility.ts

```diff
@@ -0,0 +1,74 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as Types from './types';
```

Comment:
I'm not sure if the rest of the team would agree, but I prefer explicit imports of specific named exports rather than catchalls. It's easier to see where things are being used when we want to refactor. I see the value in being able to quickly import everything when these modules are first created and start to evolve.

Reply:
Yeah this makes sense, changed!